### PR TITLE
EVSS: Add document service as subclass

### PR DESCRIPTION
**Pull request by carlmjohnson**

### CODE REVIEW

File: lib/evss/claims_service.rb [hunk 1]
```diff
@@ -1,20 +1,19 @@
 module EVSS
```
Comment (austin-martinez) [inline lib/evss/claims_service.rb]: I think now would be a good time to split this up into separate files under an `evss` folder, something like `base`, `claims`, and `documents`

**Comment by austin-martinez** [comment]
:+1: after separating files

**Comment by carlmjohnson** [comment]
Take a look at how I split it up. I'm not used to Ruby's module system, so I'm not sure if I did it correctly or not.

**Comment by austin-martinez** [comment]
@carlmjohnson What you had worked, but I don't think we need the top level evss.rb that just requires the children as we will most likely only be interfacing with a single service at a time. If we find that we are using multiple EVSS services in a single place, it would be cleaner to be able to require the whole set at once (much like if it was a gem) as you had. I also switched the require_relative to require_dependency as that is the convention we are using to allow Rails to reload any changes during development. Finally, I split the spec file as well.

**Pull request merged**